source/plugin/tab-lists.rst

```diff
@@ -0,0 +1,107 @@
+    of two separate packets for the header and the footer.
+
+Tab List Entries
+================
+
+Now that we have set the header and footer of the ``TabList``, we can also add our own entries to the list. An example
+of this is shown below:
+
+.. code-block:: java
+    
+    import org.spongepowered.api.entity.living.player.gamemode.GameModes;
+    import org.spongepowered.api.entity.living.player.tab.TabListEntry;
+    import org.spongepowered.api.profile.GameProfile;
+    
+    TabListEntry entry = TabListEntry.builder()
+        .list(tablist)
+        .gameMode(GameModes.SURVIVAL)
+        .profile(gameProfile)
+        .build();
+    tablist.addEntry(entry);
+
+Now let's break this down. We set the list associated with the ``TabListEntry`` to our specified ``TabList`` using the
+``list()`` method. We then set the game mode of our entry to ``SURVIVAL``. The game mode of our entry is used to
+determine various things. On the client, it is used to determine if a player is in creative or perhaps a spectator. If
+the game mode is spectator, then their name will also appears gray and italicized. We then need to specify the
+``GameProfile`` that the entry is associated with. The ``GameProfile`` may be constructed using the
+``GameProfile.of()`` method, or it can be obtained from a real profile, such as a player. To apply the entry to the tab
+list, we simply need to call the ``addEntry()`` method off of our ``TabList``.
+
+We can flesh out our basic example by specifying things such as the display name or latency of the entry:
+
+.. code-block:: java
+    
+    TabListEntry entry = TabListEntry.builder()
+        .list(tablist)
+        .displayName(Text.of("Spongie"))
+        .latency(0)
+        .profile(gameProfile)
+        .build();
+    tablist.addEntry(entry);
+
+Here, we set the display name that our entry will appear under to `Spongie` using the ``displayName()`` method. We then
+set the latency for our ``TabListEntry`` to five bars. See the ``TabListEntry#setLatency()`` method for more
+information on how to specify other types of bars for our entry.
+
+Modifying Current Entries
+=========================
+
+Using the ``TabList``, we can obtain entries currently on the ``TabList`` for our own modification. To obtain a
+specific entry, use the ``TabList#getEntry(UUID)`` method. This method will return ``Optional.empty()`` if the
+specified UUID cannot be found. An example is shown below:
+
+.. code-block:: java
+    
+    import java.util.Optional;
+    
+    Optional<TabListEntry> optional = tablist.getEntry(uuid);
+    if (optional.isPresent()) {
+        TabListEntry entry = optional.get(); 
+    }
+
+With this, we can use the methods on ``TabListEntry`` to modify the game mode, latency, and the display name of the
+entry:
+
+.. code-block:: java
+    
+    entry.setDisplayName(Text.of("Pretender Spongie"));
+    entry.setLatency(1000);
+    entry.setGameMode(GameModes.SPECTATOR);
```

Comment:
I would also note that you can also get all of the entries (A `Collection<TabListEntry>`) using `TabList#getEntries`, and that you can remove an entry by its associated `UUID` using `TabList#removeEntry(UUID)`. Likewise, it will return `Optional.empty()` if the UUID cannot be found.